data/sles4sap/qe_sap_deployment/mr_test_ec2.yaml

```diff
@@ -6,7 +6,7 @@ terraform:
     # GENERAL VARIABLES #
     aws_region: "%PUBLIC_CLOUD_REGION%"
     admin_user: "cloudadmin"
-    public_key : "~/.ssh/id_rsa.pub"
+    public_key: "~/.ssh/id_rsa.pub"
```

Comment:
I can see the extra space present in other conf.yaml

```
data/sles4sap/qe_sap_deployment/hanasr_azure.yaml
data/sles4sap/qe_sap_deployment/hanasr_ec2.yaml
data/sles4sap/qe_sap_deployment/hanasr_gcp.yaml
data/sles4sap/qe_sap_deployment/mr_test_azure.yaml
data/sles4sap/qe_sap_deployment/mr_test_ec2.yaml
data/sles4sap/qe_sap_deployment/sles4sap_azure_generic.yaml
```

do you like to remove it from there too?

Reply:
Done.  But I don't know why there are those changes, I really did nothing on `SAPCONF: true`.
    - SAPCONF: true
    + SAPCONF: true

Reply:
before this commit your files didn't had empty line at the end of file